rfc-brief.Rmd

```diff
@@ -0,0 +1,225 @@
+---
+title: "Data sharing specifications for the NF community"
+author:
+- name: Robert Allaway
+  url: https://example.com/
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0003-3573-3565
+- name: Jineta Banerjee
+  url: https://example.com/
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0002-1775-3645
+- name: Anh Nguyet Vu
+  url: https://github.com/anngvu
+  affiliation: Sage BioNetworks
+  affiliation_url: https://sagebionetworks.org/
+  orcid_id: 0000-0003-1488-6730
+date: "`r Sys.Date()`"
+description: |
+  Status: **Draft**
+draft: yes
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+library(gt)
+library(dplyr)
+library(tidyr)
+library(ggplot2)
+
+survey <- read.csv("materials/rfc/rfc_responses.csv")
+```
+
+## Abstract
+
+This document describes data types and formats that should be shared by the Neurofibromatosis (NF) research community for the NF Data Portal. 
+
+## Introduction
+
+The NF Data Portal is a collaborative data-sharing platform for Neurofibromatosis, a rare genetic disease. 
+It has been jointly supported by multiple funding organizations -- the Gilbert Family Foundation (GFF), Children's Tumor Foundation (CTF), Neurofibromatosis Therapeutic Acceleration Program (NTAP), NCI Developmental and Hyperactive Ras Tumor SPORE (DHART SPORE), CDMRP Neurofibromatosis Research Program (CDMRP NFRP), and Neurofibromatosis Research Initiative (NFRI) -- and therefore represents substantial NF research activity.
+
+As the NF community may continue to expand to add new affiliated or independent researchers, and therefore new data, having these standards will help maintain the quality and quantity of our valuable resources. 
+But also important is that this document should actually benefit our data contributors.
+<!-- to answer the question of "Why should I glance at this?" -->
+One advantage is allowing our contributors to prioritize submissions accordingly.
+The burden of having to submit data in multiple formats can also be avoided, since preferred formats are clear. 
+These specifications should lead to higher-impact data, leaning towards the most interoperabe formats that make data more likely to be reused (and cited). 
+Finally, this open document lets our community be apprised of the status quo, including specifications currently *not* included, so that they can continue to provide comments.
```

Comment:
```suggestion
Finally, this open document lets our community be informed of the current state of art, including specifications currently *not* included, so that they can provide input towards specific/new data formats.
```